SPECS/krb5/krb5.spec

```diff
@@ -41,11 +41,10 @@ These are the additional language files of krb5.
 %setup -q
 
 %build
-cd src &&
-sed -e 's@\^u}@^u cols 300}@' \
-    -i tests/dejagnu/config/default.exp &&
-CPPFLAGS="-D_GNU_SOURCE %{getenv:CPPFLAGS}" \
-autoconf &&
+cd src
+sed -e 's@\^u}@^u cols 300}@' -i tests/dejagnu/config/default.exp
+#autoupdate
```

Comment:
What is this comment about? Could you either remove or extend it a bit?

Reply:
Just a test - will remove